frontends/p4/def_use.cpp

```diff
@@ -932,3 +934,13 @@ bool ComputeWriteSet::preorder(const IR::MethodCallStatement* statement) {
 }
 
 }  // namespace P4
+
+// functions for calling from gdb
+void dump(const P4::StorageLocation *s) { std::cout << *s << std::endl; }
```

Comment:
If these implement IHasDbPrint there should already exist a dump, but indeed gdb sometimes has trouble finding the dump.

Reply:
Yes, gdb seems to be unable to deal with virtual base classes properly (or at least to be able to call function overloads based on them), so I ended up adding all these overloads to make them callable.